wdl/model/draft2/src/test/scala/wdl/expression/WdlStandardLibraryFunctionsSpec.scala

```diff
@@ -64,6 +65,8 @@ case object TestableFunctions extends WdlStandardLibraryFunctions {
   override def stdout(params: Seq[Try[WomValue]]): Try[WomFile] = throw new NotImplementedError
   override def globHelper(pattern: String): Seq[String] = throw new NotImplementedError
   override def stderr(params: Seq[Try[WomValue]]): Try[WomFile] = throw new NotImplementedError
+
+  override protected def fileSizeLimitationConfig: FileSizeLimitationConfig = ???
```

Comment:
This should `throw new java.lang.UnsupportedOperationException`.

1) The original meaning of `???` was "I haven't finished coding and this code is not fit to be merged yet". However `throw new NotImplementedError` means "I really want this to throw a `java.lang.Error` because this isn't implemented".
2) `throw <? extends java.lang.Error>` means "Shut down the JVM!" It's almost akin to `System.exit()` except tries to print out the error while dying and _then_ shuts down.<sup>1</sup> If something goes wrong in only one of our tests we still want to run other tests.

It's outstanding tech-debt issue to clean up our prevalent usage of `Error` including `Predef.assert()` but we shouldn't add more. 

---

<sup>1</sup> The JVM dev community has so abused [`Error`](https://docs.oracle.com/javase/8/docs/api/java/lang/Error.html) over the years that the behavior catching errors is now a whole [copypasta](https://www.scala-lang.org/api/current/scala/util/control/NonFatal$.html) of maybe-shutdown-maybe-not. 🤦‍♂️